docs/02_Planning-your-study.md

```diff
@@ -0,0 +1,27 @@
+#Planning your study
+
+When planning your study, here are some things you might want to consider:
+
+* [PLANNING YOUR STUDY]
+
+ Resources currently collected [here](/docs/reproducible_neuroimaging_research.md#Reproducible-neuroimaging-research)
```

Comment:
apparently I get a 404 error when I try this link (and with a few others)

The only way I have been able to make this work is with:

 `Resources currently collected [here](./reproducible_neuroimaging_research.md#reproducible-neuroimaging-tools)`

Reply:
I will fix this! Thanks for spotting it. Some links seem to "work" in atom, but don't on the "mkdocs serve"ed website.